tools/gbafix/gbafix.c

```diff
@@ -138,6 +138,7 @@ int main(int argc, char *argv[])
     char *argfile = 0;
     FILE *infile;
     int silent = 0;
+    int schedule_pad = 0;
```

Comment:
I would suggest upstreaming this to devkitpro.  I believe the only reason this copy exists is because there was a bugfix was needed at the time, and we didn't want to wait for another dkp release.

Reply:
Sure, if they'll take our advice.  Should I bump the version?

Reply:
They seem to bump the version every commit, so I would guess yes.

Reply:
Bumped the version number and updated the changelog.